src/main/java/com/dnastack/search/sheets/GlobalControllerExceptionHandler.java

```diff
@@ -35,6 +35,11 @@
     @ExceptionHandler(GoogleJsonResponseException.class)
     public ResponseEntity<SearchErrorResponse> handle(GoogleJsonResponseException ex) {
         log.info("Returning " + ex.getStatusCode(), ex);
+        if (ex.getStatusCode() == 401) {
```

Comment:
Do 401 statuses need to be special-cased like this, or could the authentication headers be included on any error of this type? If the 401 status does need to be special-cased, do we need to do the same for 403?

Reply:
Maybe we can discuss a bit. The (minor) "issue" here was there was no `www-authenticate` header in a 401 response. I didn't see an easy way to forward the entire header without a cast or a converter class so I gave it a special case. Is a 403 required to have the "www-authenticate" header?
![Screen Shot 2020-09-24 at 3 07 05 PM](https://user-images.githubusercontent.com/70958638/94458460-f94f4f00-0183-11eb-9cff-e79b05485a8a.png)
It's fair to claim that its a band-aid fix, and I'd be happy to discuss if this was even a true "issue" to begin with.